MAUI/Scheduler/getting-started.md

```diff
@@ -574,6 +574,25 @@ N> This property will not be applicable for the `SchedulerView` is `Timeline Mon
 
 ![show-week-number-in-maui-scheduler](images/getting-started/show-week-number-in-maui-scheduler.png)
 
+## Show Today button
+
+By Using the `ShowTodayButton` property of the [SfScheduler](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SfScheduler.html), you can show or hide the Today button. This support is to help users to show or hide the Today button dynamically irrespective to the Navigation arrows. 
```

Comment:
By Using the `ShowTodayButton` property of the [SfScheduler](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SfScheduler.html), you can show or hide the Today button. - after this explain default value and it functionality

Reply:
Updated teh changes